scripts/run-goto-transcoder.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+##############
+# PARAMETERS #
+##############
+contract_folder=target/kani_verify_std/target/x86_64-unknown-linux-gnu/debug/deps
+supported_regex=checked_unchecked.*.out
```

Comment:
Can you also make this an argument so users can easily try out their new harnesses.

Reply:
Added as an argument